### PR TITLE
FE card-2 (Registration)

### PR DESCRIPTION
**Pull request by KuzminAanton**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+    rel="stylesheet">
+  <meta
+    name="description"
+    content="Web site created using create-react-app"
+  />
+  <title>Hospital</title>
+
+</head>
+<body>
+<noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (usatovaDasha) [inline public/index.html:20]: check tabs

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+import Auth from './components/auth/Auth';
+import Main from './components/Main';
 
-function App() {
+const App = () => {
   return (
     <div className="App">
-
+      <Switch>
+        <Route path="/auth/">
```
Comment (usatovaDasha) [inline src/App.js:11]: /auth

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+          <Auth />
+        </Route>
+        <Route path="/main/">
```
Comment (usatovaDasha) [inline src/App.js:14]: /main

File: src/components/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import './Main.scss';
+
+const Main = () => {
+  const headerText = 'Приемы';
+
+  return (
+    <div>
+      <Header
+        headerText={headerText}
+        children={<Button className="header-btn">Выйти</Button>}
```
Comment (usatovaDasha) [inline src/components/Main.jsx:13]: please add correct children

File: src/components/auth/Auth.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import { Route, Switch } from 'react-router-dom';
+import Login from './Login';
+import Registration from './Registration';
+
+const Auth = () => {
+
+  return (
+    <div>
+      <Switch>
+        <Route path="/auth/login/">
```
Comment (usatovaDasha) [inline src/components/auth/Auth.jsx:11]: higher

File: src/components/auth/Registration.jsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+          setErrorAlertPass(false);
+          setErrorText('');
+          if (!(password === retryPassword)) {
+            setErrorAlertPassReplay(true);
+            setErrorAlertPass(true);
+            setErrorText('Пароли не совпадают');
+          } else {
+            setErrorAlertPassReplay(false);
+            setErrorText('');
+            console.log('AXIOSSSSS!!!!');
```
Comment (usatovaDasha) [inline src/components/auth/Registration.jsx:53]: !!!

File: src/components/auth/Registration.jsx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  TextField,
+} from '@mui/material';
+import Header from '../../elements/Header';
+import Logo from '../../elements/Logo';
+import './Auth.scss';
+
+const Registration = () => {
+  const [login, setLogin] = useState('');
+  const [password, setPassword] = useState('');
+  const [retryPassword, setRetryPassword] = useState('');
```
Comment (usatovaDasha) [inline src/components/auth/Registration.jsx:15]: why do you need so many states?

File: src/components/auth/Registration.jsx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+                name="repeatPassword"
+                type="password"
+                placeholder="Password"
+                value={retryPassword}
+                onChange={(e) => setRetryPassword(e.target.value)}
+              />
+            </div>
+          </div>
+          <div className="auth-form__btns">
+            <Button onClick={formSubmit} variant="outlined">Зарегистрироваться</Button>
```
Comment (usatovaDasha) [inline src/components/auth/Registration.jsx:116]: correct call function

File: src/components/auth/Registration.jsx [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+            <Link underline="hover" href="#">Авторизоваться</Link>
```
Comment (usatovaDasha) [inline src/components/auth/Registration.jsx:117]: no need to?

File: src/components/Main.jsx (hunk 4 above)
Comment (KuzminAanton) [reply src/components/Main.jsx:13]: this button will be correctly in next pull requests, when i can test his with `Login` functionally

File: src/components/auth/Registration.jsx (hunk 6 above)
Comment (KuzminAanton) [reply src/components/auth/Registration.jsx:53]: sorry, this `console.log` helped check validation form. this is temporarily. but? i deleted log now

File: public/index.html (hunk 1 above)
Comment (KuzminAanton) [reply public/index.html:20]: fixed

File: src/App.js (hunk 2 above)
Comment (KuzminAanton) [reply src/App.js:11]: fixed

File: src/App.js (hunk 3 above)
Comment (KuzminAanton) [reply src/App.js:14]: fixed

File: src/components/auth/Auth.jsx (hunk 5 above)
Comment (KuzminAanton) [reply src/components/auth/Auth.jsx:11]: fixed

File: src/components/auth/Registration.jsx (hunk 7 above)
Comment (KuzminAanton) [reply src/components/auth/Registration.jsx:15]: i rebuild it. input value `login`, `password` and `retryPassword` have one state. 
i make not simple error alert functional. i don't can use one state for errors. it's add more code.

File: src/components/auth/Registration.jsx (hunk 8 above)
Comment (KuzminAanton) [reply src/components/auth/Registration.jsx:116]: fixed

File: src/components/auth/Registration.jsx (hunk 9 above)
Comment (KuzminAanton) [reply src/components/auth/Registration.jsx:117]: fixed

**Pull request merged**